core/src/main/resources/static/js/envs.js

```diff
@@ -678,14 +678,24 @@ app.controller("envsCtrl", function($scope, $http, $location, $window) {
 
                         $scope.addNewCluster.type = $scope.addNewCluster.clusterType;
 
-                        if($scope.addNewCluster.host == undefined)
+                        if($scope.addNewCluster.type === 'kafkarestapi'){
+                            $scope.kafkaFlavor = 'Apache Kafka'; // this is independent of any kafka flavor type
+                        }
+
+                        if($scope.addNewCluster.host === undefined  && $scope.addNewCluster.type !== 'kafkarestapi')
```

Comment:
Braces in same line or new one, but both looks weird

Reply:
yea, updated.